src/chrome/content/rules/Jimcdn.com.xml

```diff
@@ -0,0 +1,21 @@
+
+<!--
+	Other Jimdo rulesets:
+		-Jimdo.xml
+		-Jimdo-Server.com.xml
+		-Jimstatic.com.xml
+		-Jimdo.design.xml
+-->
+<ruleset name="jimcdn.com (partial)">
```

Comment:
Please use the same case than in the filename. In this example, use `Jimcdn.com`. We also don't need the `partial` tag here.